### PR TITLE
Add send receive api

### PR DESCRIPTION
**Pull request by manq2010**
Leaderboard: hit the API
---

In these pull request, the following was done:

* [x] [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) was used
* [x] The project was built with webpack
* [x]  Plain HTML markup with minimum styling was injected to build the leaderboard wireframe
* [x]  ES6 modules, with `import` and `export` used
* [x] Used `arrow functions`
* [x] Used `async` and `await` JavaScript features to consume the API

### CODE REVIEW

File: src/modules/get-players.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  playersContainer.innerHTML = '';
+  const response = await fetch(URL);
+  const players = await response.json();
+  players.result.sort((a, b) => b.score - a.score);
+  return players.result;
+};
+
+getPlayers().catch(() => {
+
+});
```
Comment (mgmediaweb) [inline src/modules/get-players.js:17]: [OPTIONAL] Kindly, I suggest you remove the functions you are not using, that will make your code cleaner and more professional.

File: src/modules/add-player.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      const players = response.json();
+      display();
+      return players;
+    })
+    .then(() => {
+      // console.log('Success:', player.result);
+    })
+    .catch(() => {
+      // console.error('Error:', player);
+    });
```
Comment (mgmediaweb) [inline src/modules/add-player.js:28]: [OPTIONAL] Kindly, I suggest you remove the functions you are not using, that will make your code cleaner and more professional.

**Review by mgmediaweb — APPROVED** [review]
Hi @manq2010 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/add-player.js (hunk 2 above)
Comment (manq2010) [reply src/modules/add-player.js:28]: Thanks, @mgmediaweb for the feedback, it's really helpful.

**Pull request merged**